### PR TITLE
Get images with pagination loop

### PR DESCRIPTION
**Pull request by mansam**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1632849

### CODE REVIEW

File: app/models/manageiq/providers/openstack/inventory/collector/cloud_manager.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@ def host_aggregates
   def images
     return @images if @images.any?
     @images = if openstack_admin?
-                image_service.handled_list(:images, {}, true).all
+                image_service.images_with_pagination_loop
               else
                 image_service.handled_list(:images)
```
Comment (aufi) [inline app/models/manageiq/providers/openstack/inventory/collector/cloud_manager.rb]: Do we need pagination loop also in case that is_admin: false?

Comment (mansam) [reply app/models/manageiq/providers/openstack/inventory/collector/cloud_manager.rb]: When is_admin => false, `handled_list` automatically handles pagination so this should be unnecessary.

Comment (aufi) [reply app/models/manageiq/providers/openstack/inventory/collector/cloud_manager.rb]: Thanks, merging :+1:

**Comment by simaishi** [comment]
@mansam Should this be `hammer/yes` ?

**Comment by mansam** [comment]
@miq-bot add_label hammer/yes

**Comment by mansam** [comment]
@simaishi Yep sorry, label added.

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit e4bd43f898141b882374b26849b601a63b3b228d 
Author: Marek Aufart <aufi.cz@gmail.com>
Date:   Mon Oct 1 15:29:27 2018 +0200

    Merge pull request #363 from mansam/get-images-with-pagination-loop
    
    Get images with pagination loop
    
    (cherry picked from commit 4d6afd144e8c0b6c327867f9ea6e868bad281537)
    
    Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1632849
```

**Pull request merged**